lib/ansible/modules/notification/rabbitmq_publish.py

```diff
@@ -88,13 +88,32 @@
       - A dictionary of headers to post with the message.
     default: {}
     type: dict
+  cafile:
+    description:
+      - CA file used during connection to the RabbitMQ server over SSL.
+      - If this option is specified, also I(certfile) and I(keyfile) must be specified.
+    version_added: '2.10'
+  certfile:
+    description:
+      - Client certificate to establish SSL connection.
+      - If this option is specified, also I(cafile) and I(keyfile) must be specified.
+    version_added: '2.10'
+  keyfile:
+    description:
+      - Client key to establish SSL connection.
+      - If this option is specified, also I(cafile) and I(certfile) must be specified.
+    version_added: '2.10'
+
 
 
 requirements: [ pika ]
 notes:
   - This module requires the pika python library U(https://pika.readthedocs.io/).
   - Pika is a pure-Python implementation of the AMQP 0-9-1 protocol that tries to stay fairly independent of the underlying network support library.
-  - This plugin is tested against RabbitMQ. Other AMQP 0.9.1 protocol based servers may work but not tested/guaranteed.
+  - This module is tested against RabbitMQ. Other AMQP 0.9.1 protocol based servers may work but not tested/guaranteed.
+  - The certificate authentication was tested with certificates created
+    viaU(https://www.rabbitmq.com/ssl.html#automated-certificate-generation) and RabbitMQ
```

Comment:
```suggestion
    via U(https://www.rabbitmq.com/ssl.html#automated-certificate-generation) and RabbitMQ
```
Also, there recently was a discussion in #ansible-docs about the word `via`; I think the result was to avoid it, since not everyone understands it.

Reply:
@Im0 can you fix the missing space?